docs/how-tos/deploy-AGIC-with-Workload-Identity-using-helm.md

```diff
@@ -0,0 +1,88 @@
+# How to deploy AGIC via Helm using Workload Identity
+
+[[_TOC_]]
```

Comment:
github doesn't support _TOC_ unlike msft docs.

Reply:
TOC Removed from the doc